torch/csrc/jit/pybind_utils.h

```diff
@@ -354,6 +354,9 @@ inline IValue toIValue(
     case TypeKind::FloatType:
       return py::cast<double>(obj);
     case TypeKind::IntType:
+    // TODO(xintchen): Handling LayoutType and ScalarTypeType correctly.
```

Comment:
Once again, it's hard to know why this change is in here, because as-is this PR doesn't ever use LayoutType or ScalarTypeType anywhere. As such, it's not self-contained enough to review properly.